src/server/service/diagnostics/log.rs

```diff
@@ -355,10 +377,9 @@ mod tests {
     }
 
     fn timestamp(time: &str) -> i64 {
-        match DateTime::parse_from_str(time, "%Y/%m/%d %H:%M:%S%.3f %z") {
-            Ok(t) => t.timestamp_millis(),
-            Err(_) => INVALID_TIMESTAMP,
-        }
+        DateTime::parse_from_str(time, "%Y/%m/%d %H:%M:%S%.3f %z")
+            .unwrap()
```

Comment:
What will happen if the time format is invalid?

Reply:
Will crash. This method is only for test. In current test cases, there are no invalid time format cases. If some invalid time format cases added later, we need to change this method correspondingly.

Reply:
If this function is test only, I recommend to mark a `#[cfg(test)]` for it to avoid accidentally use it.

Reply:
It is a util method for test cases and already inside the `#[cfg(test)]`.